dashboard/client/src/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -52,25 +52,41 @@ class BreadcrumbsWithRouter extends Component {
       });
     });
   }
+
   componentWillUnmount() {
     this.unlisten();
   }
+
   render() {
     const { parts } = this.state;
 
-    const links = parts.map((p, i) => {
-      const isActive = i === parts.length - 1;
+    const links = parts.map(({ path, name }, i) => {
+      const itemProps = {};
+      let itemContent = (
+        <NavLink to={path}>
+          { name }
+        </NavLink>
+      );
+
+      if (i === parts.length - 1) {
+        itemProps.active = true;
+
+        itemContent = name;
+      }
+
       return (
-        <li key={i} className={isActive ? 'active' : undefined}>
-          {isActive ? p.name : <NavLink to={p.path}>{p.name}</NavLink>}
-        </li>
+        <BreadcrumbItem key={`${name}${i}`} {...itemProps}>
+          { itemContent }
+        </BreadcrumbItem>
       );
     });
 
-    let breadcrumbClassName = 'breadcrumb';
-
-    return <ol className={breadcrumbClassName}>{links}</ol>;
+    return (
+      <Breadcrumb listClassName="background-color-inherit-important m-0">
+        { links }
+      </Breadcrumb>
+    );
   }
 }
 
-export const Breadcrumbs = withRouter(BreadcrumbsWithRouter);
```

Comment:
Why did we drop the name?

Reply:
Because we are exporting it as a default component